index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
- [ ] It would be better if we format our PR nicely to include details of our work. For example;

```
In this project, I did the following;
- Setup linters
- Maintain a professional Repository
- Github flow used correctly
- Necessary files created

```

Reply:
is everything fine now?